generation/image_to_image_translation/tutorial_segmentation_with_ddpm.ipynb

```diff
@@ -0,0 +1,1317 @@
+{
```

Comment:
<p>“## Preprocessing of the BRATS Dataset in 2D slices for training”.</p><p> </p><p>The first line has one more&nbsp;"\"</p><p> </p><p>Looks the description here is inconsistent with transforms below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1781/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>